### PR TITLE
solving issue: #837 overflow problem of bigger links in discussion thread

### PR DESCRIPTION
**Pull request by piyushmittal25**
to solve the overflow problem of long URLs on discussion threads
Fixes #\<837\>

### CODE REVIEW

**Comment by piyushmittal25** [comment]
It's my first time so please help me

File: client/css/_discussion.scss [hunk 1]
```diff
@@ -55,3 +55,6 @@
 .preview-pane {
   padding: 6px 12px;
 }
+p{
```
[the remark this answers is not in the document]
Comment (piyushmittal25) [reply client/css/_discussion.scss]: @wuworkshop  i have changed now you can review that again.

[the remark this answers is not in the document]
Comment (piyushmittal25) [reply client/css/_discussion.scss]: @wuworkshop i have changed that now you can review that again:)

**Comment by piyushmittal25** [comment]
@lpatmo sorry for that actually this is my first open source contribution so i didn't know about the labels:)

**Comment by nalbina** [comment]
and hooray @piyushmittal25 first PR 🎉

**Pull request merged**